activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -22,6 +22,28 @@ module ActiveSupport
   #   crypt = ActiveSupport::MessageEncryptor.new(key)                            # => #<ActiveSupport::MessageEncryptor ...>
   #   encrypted_data = crypt.encrypt_and_sign('my secret data')                   # => "NlFBTTMwOUV5UlA1QlNEN2xkY2d6eThYWWh..."
   #   crypt.decrypt_and_verify(encrypted_data)                                    # => "my secret data"
+  #
+  # MessageEncryptor supports the following metadata that's associated with a message:
+  #
+  # * <tt>+:purpose+</tt> - The purpose for which the message is intended to be used.
+  # * <tt>+:expires_at+</tt> - The expiration date of the message.
+  # * <tt>+:expires_in+</tt> - The relative expiration date. Note that, :expires_at takes
```

Comment:
Whoops, sorry, my comment wasn't clear. Actually, RDoc allows you to write formatted code snippet through `<tt>code</tt>` or with `+code+`. Though, the latter, while simpler, doesn't support a lot of fashions.

My original comment  was about the `:expires_at` and `:expires_in` references in this sentence.